students/Sigul4/hw-6+7/task-1.js

```diff
@@ -0,0 +1,19 @@
+// ********************
+// 
+// Severability reaction
+// 
+// ********************
+
+const NumberToCheck = Math.floor(3);
```

Comment:
Please provide variations and functions naming with `camelCase`.
`PascalCase` is reserved for classes.